test/externalTests/gp2.sh

```diff
@@ -67,11 +67,11 @@ function gp2_test
     force_hardhat_compiler_binary "$config_file" "$BINARY_TYPE" "$BINARY_PATH"
     force_hardhat_compiler_settings "$config_file" "$(first_word "$SELECTED_PRESETS")" "$config_var"
     force_hardhat_unlimited_contract_size "$config_file" "$config_var"
-    npm install
+    yarn
 
     # New hardhat release breaks GP2 tests, and since GP2 repository has been archived, we are pinning hardhat
     # to the previous stable version. See https://github.com/ethereum/solidity/pull/13485
-    npm install hardhat@2.10.2
+    yarn add hardhat@2.10.2
```

Comment:
Can you try to remove this workaround? You switched to `cowprotocol/contracts` and that might have the original issue (#13485) fixed.

Reply:
No, the issue still present in the `cowprotocol/contracts`. So we still need the workaround for now.

Reply:
Fine. Thanks for checking at least :)